packages/access-api/src/bindings.d.ts

```diff
@@ -53,6 +49,11 @@ export interface Env {
   SPACES: KVNamespace
   VALIDATIONS: KVNamespace
   W3ACCESS_METRICS: AnalyticsEngine
+  /**
+   * primary r2 bucket used by access-api.
+   * will be used for storing env.models.delegations CARs
+   */
+  ACCESS_API_R2: R2Bucket
```

Comment:
Just have a bucket specific to the things it's storing? We can create many of these - we don't have to have only 1 for everything we store.

Reply:
https://github.com/web3-storage/w3protocol/pull/603